src/main/kotlin/no/nav/hjelpemidler/delbestilling/delbestilling/DelbestillingService.kt

```diff
@@ -138,7 +138,16 @@ class DelbestillingService(
 
         sendStatistikk(request.delbestilling, utlån.fnr)
 
-        return DelbestillingResultat(id, null, saksnummer = lagretSaksnummer)
+        // Hent ut den nye delbestillingen
+        val delbestillingSak = if (lagretSaksnummer != null) {
+            delbestillingRepository.withTransaction { tx ->
+                delbestillingRepository.hentDelbestilling(tx, lagretSaksnummer)
+            }
```

Comment:
Dersom vi endrar hentDelbestilling() til å bruke ein default session når ein ikkje sender inn ein transaksjon https://github.com/navikt/hm-delbestilling-api/blob/main/src/main/kotlin/no/nav/hjelpemidler/delbestilling/delbestilling/DelbestillingRepository.kt#L78 
slik:
```
fun hentDelbestilling(saksnummer: Long, tx: Session = sessionOf(ds)): DelbestillingSak? = tx.run(
```
så trur eg vi kan fjerne transaksjonen her.
```suggestion
            delbestillingRepository.hentDelbestilling(tx, lagretSaksnummer)
```